### ISSUE TITLE
Content from html should be in README.md

### ISSUE THREAD

**Issue by nicksuch** [ChaChaSlide/ChaChaSlide.github.io#1]
Nice work getting all your links into the `html` file, and getting it all pushed to GitHub! I noticed that you also created a `README.md` file. This is where we'd like to see the content that you currently have in the `html` file. Could you move it there?

Once you get that content moved, another thing you could improve is your formatting for your content. Check out how I did [my README](https://github.com/nicksuch/nicksuch.github.io/blob/master/README.md) as an example (clicking the `Raw` button may help you see my actual code), and see the [Markdown Syntax reference](https://daringfireball.net/projects/markdown/syntax), specifically for Lists and Links.

**Comment by ChaChaSlide** [comment]
Yes sir, I have the read.me file edited but when I push it. It doesn't add
the changes to it. That's why I made the html file in the first place.

On May 3, 2017 3:49 PM, "Nick Such" <notifications@github.com> wrote:

Nice work getting all your links into the html file, and getting it all
pushed to GitHub! I noticed that you also created a README.md file. This is
where we'd like to see the content that you currently have in the html
file. Could you move it there?

Once you get that content moved, another thing you could improve is your
formatting for your content. Check out how I did my README
<https://github.com/nicksuch/nicksuch.github.io/blob/master/README.md> as
an example (clicking the Raw button may help you see my actual code), and
see the Markdown Syntax reference
<https://daringfireball.net/projects/markdown/syntax>, specifically for
Lists and Links.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/ChaChaSlide/ChaChaSlide.github.io/issues/1>, or mute
the thread

**Comment by nicksuch** [comment]
Got it. I think I understand the issue better now. I'll send you a message on Slack with some more questions to help resolve it, or we can set up a time to meet and discuss!

**Comment by ChaChaSlide** [comment]
I tried creating a new repo last night and I still can't push the changes.
When I cat the file or prints out the contents correctly. Not sure why I
can't push it.

**Comment by ChaChaSlide** [comment]
Hey Nick, I've set my html code to the master branch but it still pulls the
the rep instead of the html code. What else should I try to fix it?

On May 4, 2017 10:13 AM, "Cha'Sed Miller" <chachaslide20@gmail.com> wrote:

I tried creating a new repo last night and I still can't push the changes.
When I cat the file or prints out the contents correctly. Not sure why I
can't push it.
